src/Microsoft.IdentityModel.Tokens/Exceptions/SecurityTokenMalformedTokenException.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.Serialization;
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    /// <summary>
+    /// Represents a <see cref="SecurityToken"/> exception when the token is malformed.
+    /// </summary>
+    [Serializable]
+    public class SecurityTokenMalformedTokenException : SecurityTokenArgumentException
```

Comment:
>SecurityTokenMalformedTokenException [](http://example.com/codeflow?start=17&length=36)

I think the name is a bit redundant including the word token twice.

Could we rename it to something like MalformedSecurityTokenException?